R/plotNorm.R

```diff
@@ -7,7 +7,7 @@
 #' Normalization is performed by edgeR::calcNormFactors. Note TMM is specifically tailored to count-based
 #' data.  Thus this function is only appropriate for count-based data.
 #'
-#' @param dat  A DGEobj or counts matrix.
+#' @param DGEdata  A DGEobj or counts matrix.
```

Comment:
@kar-agg-gen This is exactly what we're looking for - changing the really generic input names to something more meaningful.